### PR TITLE
Update SubWallet and Dotinsights info

### PR DESCRIPTION
**Pull request by djhatrang**
1. Update SubWallet
- Description & Treasury Proposal
- Features
- Wallet order in extension & mobile tables. 

*Note:
- We prioritize native & active wallets (SubWallet, Nova Wallet & Talisman) in the ecosystem with the most features/functions, and rank according to tractions (SubWallet has 41K+ installs on both extension & mobile app). Multi-chain wallet like Enkrypt supports very few networks & does not offer a lot of functions, so should not be on top.

2. Update Dotinsights
- Polkadot & Kusama Ecosystem Map 
- Polkadot Deep Dive Report

### CODE REVIEW

File: docs/general/wallets.md [hunk 1]
```diff
@@ -41,21 +41,23 @@ Browser extension wallets at bare minimum act as key storage and management solu
 
 | Wallet                                | Browsers                                    | Staking and Nomination Pools | NFTs    | Crowdloans | Ledger support | Governance | Other features                   |
 | ------------------------------------- | ------------------------------------------- | ---------------------------- | ------- | ---------- | -------------- | ---------- | -------------------------------- |
-| [Enkrypt](https://www.enkrypt.com/)   | Brave, Chrome, Edge, Firefox, Opera, Safari | No, No                       | **Yes** | No         | **Yes**        | No         | [Enkrypt Features](#enkrypt)     |
```
Comment (profradha) [inline docs/general/wallets.md]: The wallets are placed in alphabetical order. I request you to revert this commit and make changes to the columns, if there is any missing information.

**Review by DrW3RK — CHANGES_REQUESTED** [review]

**Comment by DrW3RK** [comment]
> We prioritize native & active wallets (SubWallet, Nova Wallet & Talisman) in the ecosystem with the most features/functions, and rank according to tractions (SubWallet has 41K+ installs on both extension & mobile app). Multi-chain wallet like Enkrypt supports very few networks & does not offer a lot of functions, so should not be on top.

The list is sorted in alphabetical order, presenting the user with the essential functionalities, and it is not a ranked list (We can add a note explaining the same on the Wiki). We look forward to making this an unbiased, uncontroversial resource that is resourceful for a typical user in the ecosystem. If you think of additional columns to ensure the user makes an informed choice, please suggest them, and we shall add them to the table.

**Comment by djhatrang** [comment]
Yes, I was about to say that a note explaining the alphabetical order would be helpful.

Some of the columns that the team can consider adding are:
- Number of supported networks
- QR-signer support
- In-app XCM Transfer

**Comment by DrW3RK** [comment]
We will create a different PR and tag you for review. For now, please revert the changes to the wallets page in this PR and will merge the changes you made to the community page.

**Comment by djhatrang** [comment]
Done! Thank you for the prompt feedback :")

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**